### PR TITLE
Fix Task List styles

### PR DESCRIPTION
**Pull request by psealock**
Partially solves https://github.com/woocommerce/woocommerce-admin/issues/3897

* Purple icons in the task list should be `$blue-medium-focus`.
* Snackbar styling and positioning should match Gutenberg.
* Image upload button (add a logo task) should match the button in the Image block (https://cloudup.com/cmauCa__tYQ).

### Screenshots

Screen Shot 2020-06-16 at 11 19 41 AM
Screen Shot 2020-06-16 at 11 33 48 AM
Screen Shot 2020-06-16 at 11 34 15 AM
Screen Shot 2020-06-16 at 11 57 46 AM

### Detailed test instructions:

1. Enable Task List.
2. Make sure the styling looks OK, including the steps themselves.
2. Confirm items found in the bullets above.
2. Do an action to fire a snackbar confirmation. See the new position is the same as the editor.
3. See links should be default blue.

### Changelog Note:

no changelog note.

### CODE REVIEW

**Comment by psealock** [comment]
Blocked due to needing to know if the Task List gets studio pink buttons like the OBW or default g2 colors.

**Comment by jameskoster** [comment]
None of the buttons should be pink anywhere :D

File: client/layout/transient-notices/style.scss [hunk 1]
```diff
@@ -3,15 +3,14 @@
 .woocommerce-transient-notices {
 	position: fixed;
 	bottom: $gap-small;
-	left: 0;
+	left: 176px;
```
Comment (becdetat) [inline client/layout/transient-notices/style.scss:6]: This seems to have affected the OBW as well, is that intentional?

Comment (becdetat) [reply client/layout/transient-notices/style.scss:6]: And when the menu is collapsed:
image
Fixing these would be a little tricky though, they're a little edge-casey, and aren't a big deal so you could probably leave them for a follow-up.

Comment (psealock) [reply client/layout/transient-notices/style.scss:6]: I think so, but this was under the Task List section of the issue.

>Snackbar styling and positioning should match Gutenberg.

I think its OK, we can always revert if need be.

Comment (becdetat) [reply client/layout/transient-notices/style.scss:6]: Cool

**Pull request merged**